config/views.yaml

```diff
@@ -29,6 +29,7 @@ component_readiness:
           - amd64
         FeatureSet:
           - default
+          - techpreview
```

Comment:
Wondered if we should include it by default for sno, other views but it isn't the only option that only applies to main..